packages/engine/src/scene/functions/loaders/ModelFunctions.ts

```diff
@@ -17,3 +17,11 @@ export const deserializeModel: ComponentDeserializeFunction = (
    */
   if (!getState(EngineState).sceneLoaded.value) setComponent(entity, SceneAssetPendingTagComponent, true)
 }
+
+export const serializeModel: ComponentSerializeFunction = (entity) => {
```

Comment:
adding this back is unnecessary with the change of getComponent to serializeComponent in GLTFConversion